packages/hawtio/src/plugins/camel/context.ts

```diff
@@ -51,10 +71,9 @@ export function useCamelTree() {
       setLoaded(true)
     }
 
-    const listener = () => {
-      setSelectedNode(null)
+    const listener = async () => {
       setLoaded(false)
-      loadTree()
+      await loadTree()
```

Comment:
`EventListener` is not `() => Promise<void>` so it doesn't take any effect to use async/await here. You can simply call `loadTree()` without async.